binding.cc

```diff
@@ -121,7 +121,7 @@ class Encoder : public Nan::ObjectWrap
 
     // Build cookie buffer.
     uint32_t cookieSize = e->enc_.GetMagicCookieSize(e->outf_.mChannelsPerFrame);
-    char cookie[cookieSize];
+	char* cookie = (char*)alloca(sizeof(char) * (cookieSize));
```

Comment:
The code should have the same effect on gcc, the alloca statement is just more specific. One would need to check if it truly gives the same code path, but I feel preprocessor directives are an "ugly" hack and should be minimized. I already hated the fact I needed to add one for __inline.